.eslintrc.js

```diff
@@ -39,8 +42,48 @@ module.exports = {
         project: './tsconfig.json',
       },
     },
+    {
+      /**
+       * Turn off sort-keys for generated constants file (we don't want to sort because of comments, but want to do other linting)
+       */
+      files: ['src/styles/constants/*.ts'],
+      rules: {
+        'sort-destructure-keys/sort-destructure-keys': 'off',
+        'sort-keys': 'off',
+        'sort-keys/sort-keys-fix': 'off',
+        'typescript-custom-sort-keys/interface': 'off',
+      },
+    }
```

Comment:
Just for record. We shouldn't change the order of generated constant file, because it will make this will mess up with the section comments.